http/handler.go

```diff
@@ -132,6 +132,7 @@ func Handler(props *vault.HandlerProperties) http.Handler {
 		mux.Handle("/v1/sys/unseal", handleSysUnseal(core))
 		mux.Handle("/v1/sys/leader", handleSysLeader(core))
 		mux.Handle("/v1/sys/health", handleSysHealth(core))
+		mux.Handle("/v1/sys/monitor", handleSysMonitor(core))
```

Comment:
One thing to think about is that since this is being defined like this it is not going through Vault's ACL system and is therefore unauthenticated. I don't believe this should be the case. I think we need to move the logic into the logical system backend.

Reply:
Ah, thanks for pointing that out.  I didn't realize that was the case.  I'll take a look at the logical system backend.